### ISSUE TITLE
Convective below-cloud impaction is partitioned by the stratiform cloud fraction, the only cover WetScavenging has

### ISSUE THREAD

**Issue by duncanwp** [climate-analytics-lab/jax-gcm#781]
Split out of the review of #775 (nit 6 there); pre-existing, not a regression from that PR.

`WetScavenging` partitions the grid box for below-cloud impaction with

```python
clear_fraction = jnp.clip(1.0 - cloud_fraction, 0.0, 1.0)
```

in `below_cloud_rate` (`jcm/physics/aerosol/jam/wetdep/wetdep_term.py`), where `cloud_fraction` is the **stratiform** cover from `CloudData`. That is the correct partition for the stratiform carrier. It is applied unchanged to the **convective** carrier, which has no cover of its own in this term.

Consequence: in a deep convective column with little stratiform cloud, `clear_fraction ≈ 1`, so essentially the full grid-mean interstitial aerosol is exposed to convective impaction — through the depth of the convective cloud as well as below its base — on top of the in-plume sink that `ConvectiveTracerTransport` already applies with `in_plume_convective=True`. The aerosol inside the plume is being removed twice under two different area assumptions.

This predates #775: the mask that PR removed (`p_full >= p_conv_top`) also spanned the cloud depth, so the exposure is the same. What changed is that the stratiform `cf` is now the *sole* partition, which is why it is worth writing down.

**Reference formulation.** ECHAM `xt_wetdep` / HAM `mo_ham_wetdep` partition the box with the cover belonging to the carrier that is doing the scavenging, and Tiedtke carries a convective cloud cover for exactly this purpose. The hooks are present: `ConvectionData` already publishes `qc_conv`/`qi_conv`, `precip_flux` and `ktype`, so a convective area fraction (or the `zcucov = 0.05` fractional precip cover `cuflx` already uses for sub-cloud evaporation, `flux_tendencies.py`) could partition the convective pathway without new state.

Suggested shape: give `below_cloud_rate` the carrier's own cover, pass the stratiform `cf` for the stratiform flux and a convective cover for `precip_flux`, and make the in-plume/environment split explicit so the two sinks do not both act on the same air.

**Comment by duncanwp** [comment]
Closing as already fixed on dev: PR #776 (merged 2026-09-14) reworked below-cloud scavenging so the convective carrier uses its own HAMMOZ updraft-area cover (M_u/(rho*w_u), prep_wetdep_hydro form) via conv_below_cloud_rate, and the stratiform 1-cloud_fraction partition no longer touches the convective path at all — see the PR's section 6 and the regression test wetdep_test.py::test_conv_washout_acts_in_the_updraft_footprint. The 'Closes #781' keyword in #776 was evidently added after merge, so the issue never auto-closed. PR #863 subsequently unified the cover behind the shared updraft_area_cover helper. The one deferred refinement — publishing the cover from ConvectionData with the true updraft density instead of the environment-density stand-in (<2% effect) — is recorded in #863's PR body/design doc as parked pending its own A/B, and is deliberately not re-filed here.
